config/manager/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:latest
+      - image: localhost:5000/ci-e2e/cluster-api-azure-controller-amd64:20201022010025
```

Comment:
these changes look like they pushed by accident

Reply:
cc @devigned do you think we should at least add a check in make verify to fail if this content has changed?

Reply:
Good catch!

I support anything that makes it harder to commit unintended changes.

I'd really like to get in a PR to remove the code manipulating these files.

Reply:
👍  Yep, I have a revert commit ready to push after I test something else.